src/tokens-studio/spectrum2-colors/spectrum2/component/light.json

```diff
@@ -126,6 +126,64 @@
         }
       }
     },
+    "menu-item": {
+      "background-color": {
+        "default": {
+          "value": "{Palette.gray.100}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "menu-item-background-color-default"
+            }
+          }
+        },
+        "hover": {
+          "value": "{Palette.gray.100}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "menu-item-background-color-hover"
+            }
+          }
+        },
+        "down": {
+          "value": "{Palette.gray.100}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "menu-item-background-color-down"
+            }
+          }
+        },
+        "key-focus": {
+          "value": "{Palette.gray.100}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "menu-item-background-color-keyboard-focus"
+            }
+          }
+        },
+        "disabled": {
+          "value": "{Palette.gray.100}",
+          "type": "color",
+          "$extensions": {
+            "spectrum-tokens": {
+              "uuid": "",
+              "name": "menu-item-background-color-disabled"
+            }
+          }
+        }
+      },
+      "background-opacity": {
+        "value": "0",
+        "type": "opacity"
```

Comment:
@nabuhasan The opacity token should be added to the "non-color" set. We talked about this yesterday...